composeApp/src/commonMain/composeResources/values/strings-common.xml

```diff
@@ -58,6 +58,12 @@
     <string name="TestResults_NotAvailable">N/A</string>
     <string name="TestResults_Summary_Performance_Hero_Upload">Upload</string>
     <string name="TestResults_Summary_Performance_Hero_Download">Download</string>
+
+
+    <string name="TestResults_Summary_Websites_Hero_Tested_Singular">Tested</string>
+    <string name="TestResults_Summary_Websites_Hero_Blocked_Singular">Blocked</string>
+    <string name="TestResults_Summary_Websites_Hero_Reachable_Singular">Accessible</string>
```

Comment:
There are different strings for circunvention, websites and instant messaging

Reply:
Its thesame string. I doubt we want to keep 12+ strings saying similar things.